backend/organization/utility/project.py

```diff
@@ -60,6 +70,45 @@ def create_new_project(data: Dict, source_language: Language) -> Project:
     project.save()
     return project
 
+<<<<<<< HEAD
+def add_project_member(project,user,user_role,role_in_project,availability):
+    """
+    Adds a user to a project. Assumes valid data at input.
+
+
+    """
+
+    ProjectMember.objects.create(
+                        project=project
+                        ,user=user
+                        ,role=user_role
+                        ,role_in_project=role_in_project
+                        ,availability=availability
+                    )
+    return 
+
+
+def get_project_admin_creators(project,limit_to_admins=False):
+    """
+    Returns a given project UserProfiles of Creators or Administrators. if limit_to_admins is set to True, only admins will be returned.
+    :param project: target project 
+    :type project: Project 
+    :param limit_to_admins: limit output to admins only 
+    :ype limit_to_amins: bool
+    """
+    targets_roles = Role.objects.filter(Q(name="Creator") | Q(name="Administrator")).all()
+    if targets_roles.count() < 1: raise Exception(f"Project does not have any Admins! {targets_roles}")  
```

Comment:
This actually gets thrown if there is no creator or administrator role in the database, not if the project has no admins (If I interpret it correctly)